api/sheets/get_dcid.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Get place DCID
+nav_order: 4
+parent: REST
+grand_parent: API
+---
+
```

Comment:
This seems very similar to find_dcid. May point out the difference?

Reply:
Sorry, I renamed find_dcid to get_dcid but I didn't delete the original in the commit. I deleted it now.